apis/extension/cpu_normalization.go

```diff
@@ -35,6 +35,14 @@ const (
 
 	// AnnotationCPUBasicInfo denotes the basic CPU info of the node.
 	AnnotationCPUBasicInfo = NodeDomainPrefix + "/cpu-basic-info"
+
+	// NormalizationRatioDiffEpsilon is the min difference between two cpu normalization ratios.
+	NormalizationRatioDiffEpsilon = 0.01
+
+	// AnnotationOriginalNodeAllocable denotes the original node allocatable, the value is a json string of corev1.ResourceList.
+	// e.g. {"cpu":"1000m"}.
+	// NOTE: contains only cpu-related fields.
+	AnnotationOriginalNodeAllocable = NodeDomainPrefix + "/original-node-allocatable"
```

Comment:
We already have the `AnnotationNodeRawAllocatable` for this.

Reply:
After seeing this definition, I will adjust it.